src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -13993,10 +13996,9 @@ void Assembler::orq(Register dst, Register src) {
 void Assembler::popcntq(Register dst, Address src) {
   assert(VM_Version::supports_popcnt(), "must support");
   InstructionMark im(this);
-  emit_int32((unsigned char)0xF3,
-             get_prefixq(src, dst),
-             0x0F,
-             (unsigned char)0xB8);
+  emit_int8((unsigned char)0xF3);
+  prefixq(src, dst, true /* is_map1 */);
+  emit_int8((unsigned char)0xB8);
```

Comment:
Just a nit, this could be:
emit_prefix_and_int8(get_prefixq(src, dst, true /* is_map1 */), (unsigned char) 0xB8);

Reply:
Thanks, done.